proforma/proforma_generator.py

```diff
@@ -518,93 +518,44 @@ def fill_alignment(ws, series, current_row, alignment):
                            number_format='##%')
     current_row += 1
     ws['A{}'.format(current_row)] = "Year one total site carbon dioxide emissions (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = site.year_one_emissions_lb_C02
+    ws['B{}'.format(current_row)] = site.year_one_emissions_tCO2
     make_attribute_row(ws, current_row)
     current_row += 1
 
     ws['A{}'.format(current_row)] = "Year one total site carbon dioxide emissions BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = site.year_one_emissions_bau_lb_C02
+    ws['B{}'.format(current_row)] = site.year_one_emissions_tCO2_bau
     make_attribute_row(ws, current_row)
     current_row += 1
     
     ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from electric utility purchases (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_lb_C02
+    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_tCO2
     make_attribute_row(ws, current_row)
     current_row += 1
     
     ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from electric utility purchases BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_bau_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-    
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from backup generator use (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = generator.year_one_emissions_lb_C02
```

Comment:
@Bill-Becker if you want to replace the onsite fuel emissions outputs, they are now aggregated across all techs rather than listing each tech: site.year_one_emissions_from_fuelburn_tCO2, site.year_one_emissions_from_fuelburn_tCO2_bau, and same for NOx, SO2, PM2.5. Could also do this down the road instead of in this PR